src/js/containers/award/AwardContainer.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+  * AwardContainer.jsx
+  * Created by Emily Gullo 01/19/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Award from 'components/award/Award';
+
+import * as SearchHelper from 'helpers/searchHelper';
+import * as awardActions from 'redux/actions/award/awardActions';
+
+const propTypes = {
+    setSelectedAward: React.PropTypes.func,
+    params: React.PropTypes.object
+};
+
+class AwardContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.getSelectedAward = this.getSelectedAward.bind(this);
+    }
+
+    getSelectedAward() {
+        const input = this.props.params.awardId;
+
+        if (this.selectedAwardRequest) {
+            // A request is currently in-flight, cancel it
+            this.selectedAwardRequest.cancel();
+        }
+
+        const selectedAwardParams = {
+            filters: [
+                {
+                    field: "piid",
+                    value: input,
+                    operation: "equals"
+                }
+            ]
+        };
+
+        this.selectedAwardRequest = SearchHelper.fetchAward(selectedAwardParams);
```

Comment:
As discussed in person, the filter parameters may be best generated by a SearchOperation model so as to keep query logic centralized.

Additionally, consider switching the API response to being parsed into a standardized Award model object rather than directly storing the JSON output to Redux.